portfolio/src/main/webapp/index.html

```diff
@@ -1,56 +1,54 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
+    s<meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-        <div id = "nav_bar">
+        <div id = "nav-bar">
             <button >Home</button>
-            <div id="scroll_buttons">
-            <button>About</button>
-
-            <button>Experience</button>
-            <button>Education</button>
-            <button>Projects</button>
-            <button>Interests</button>
+            <div id="scroll-buttons">
+            <button id="about-button" onclick="moveToAbout()">About</button>
```

Comment:
You still have indentation issues here btw. Check my prior [comment](https://github.com/nourishnew/software-product-sprint-portfolio-ncherish/pull/2#discussion_r891816797) to see how to address.